src/settings/components/OaiNotification.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import css from './OaiNotification.css';
+
+export default class OaiNotification extends Component {
+  static propTypes = {
+    isOaiServiceEnabled: PropTypes.bool.isRequired,
+  };
+
+  render() {
+    const {
+      isOaiServiceEnabled,
+    } = this.props;
+
+    return (
+      isOaiServiceEnabled === false && (
```

Comment:
```suggestion
      !isOaiServiceEnabled && (
```
I would suggest the above for brevity. As for me, the main idea is to be consistent with the chosen approach later on.

Reply:
Fixed